_fossfund/18-msys2.md

```diff
@@ -0,0 +1,10 @@
+---
+title: MSYS2
+logo: msys2.svg
```

Comment:
this [image](https://avatars.githubusercontent.com/u/6759993?s=200&v=4), should be added to https://github.com/microsoft/opensource.microsoft.com/tree/1dbc9603ccbda0f41c0535267ca502abe80aa0aa/assets/images/fossfund